applications/plugins/Geomagic/src/GeomagicDriver.cpp

```diff
@@ -65,18 +87,18 @@ using namespace sofa::defaulttype;
 
 void printError(const HDErrorInfo *error, const char *message)
 {
-    std::cerr << hdGetErrorString(error->errorCode) << std::endl;
-    std::cerr << "HHD: "<< error->hHD << std::endl;
-    std::cerr << "Error Code: "<< error->hHD << std::endl;
-    std::cerr << "Internal Error Code: "<< error->internalErrorCode<<std::endl;
-    std::cerr << "Message: " << message << std::endl;
+    msg_error("GeomagicDriver") << hdGetErrorString(error->errorCode);
+    msg_error("GeomagicDriver") << "HHD: "<< error->hHD;
+    msg_error("GeomagicDriver") << "Error Code: "<< error->hHD;
+    msg_error("GeomagicDriver") << "Internal Error Code: "<< error->internalErrorCode;
+    msg_error("GeomagicDriver") << "Message: " << message;
```

Comment:
`msg_error()` is enough, GeomagicDriver inherits from Base.

Reply:
done for the other use case in the class.
But this one is a static method used on a dedicated thread. So the "GeomagicDriver" should stay.